src/py/flwr/common/logger.py

```diff
@@ -185,10 +203,6 @@ def configure(
         FLOWER_LOGGER.addHandler(http_handler)
 
 
-logger = logging.getLogger(LOGGER_NAME)  # pylint: disable=invalid-name
-log = logger.log  # pylint: disable=invalid-name
```

Comment:
Why are these removed? IIRC, at least `log` is required b/c it's exported via `flwr.common.__init__`

Reply:
We moved the `log = logger.log` to the top of the file.